php/media/class-global-transformations.php

```diff
@@ -464,6 +464,9 @@ private function init_taxonomy_manager( $post ) {
 		}
 
 		$out[] = '</div>';
+		if ( ! function_exists( 'use_block_editor_for_post' ) || ! use_block_editor_for_post( $post ) ) {
+			wp_enqueue_script( 'term-ordering', $this->media->plugin->dir_url . '/js/classic-editor.js', array(), $this->media->plugin->version, true );
```

Comment:
@DavidCramer 
Since we are changing the script variable name inside, from `TermsOrder` to something more abstract like `ClassicEditor`, perhaps we can find a more abstract slug?

Reply:
@pereirinha - Good idea.